esp-hal/Cargo.toml

```diff
@@ -111,23 +111,25 @@ rt = [
 ]
 ## Enable interrupt vectoring.
 vectored = ["procmacros/interrupt"]
+## Configuration for placing device drivers in the IRAM for faster access.
+place-spi-driver-in-ram = []
 
-#! ### Chip Support Feature Flags
```

Comment:
I removed these characters so they don't show up on the top level docs, because I felt it was just noise - but I understand this might be a bit controversial so opinions welcome :).

Reply:
I think it looks better but I don't have a strong opinion here

Reply:
Yeah that's probably fine, I had just documented all the features initially but now that our documentation is sorted it probably doesn't make sense to display those features.